recipes/pyexpander/meta.yaml

```
@@ -0,0 +1,39 @@
+{% set name = "pyexpander" %}
+{% set version = "1.8.4" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: "27885ee926cb05bae3fb03e7572837b7233b41cd4c3f7bbc6cb9c099ccaefbcd"
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv "
```

No need for these arguments anymore, they're there by default.
```suggestion
  script: "{{ PYTHON }} -m pip install . -vv "
```

---

Done